cms/djangoapps/contentstore/views/item.py

```diff
@@ -1230,6 +1230,11 @@ def create_xblock_info(xblock, data=None, metadata=None, include_ancestor_info=F
                     proctoring_exam_configuration_link = get_exam_configuration_dashboard_url(
                         course.id, xblock_info['id'])
 
+                if course.proctoring_provider == 'proctortrack':
```

Comment:
This approach is less generic than the one we discussed; if we can plug the name of the proctoring provider in as part of the flag name we check, then our code reflects less of the ad-hoc specificity of the situation we're in. That sort of specificity we want to push into configuration as much as possible (especially given the existence of Open edX instances).

This comment is non-blocking feedback.

Reply:
Yeah, this isn't awesome.